documentation/examples/prometheus-service.yml

```diff
@@ -43,7 +43,7 @@ items:
         serviceAccountName: prometheus
         containers:
         - name: prometheus
-          image: gcr.io/prometheus-to-sd/stackdriver-prometheus:master
+          image: gcr.io/stackdriver-prometheus/stackdriver-prometheus:release-0.4.2
```

Comment:
I've been keeping the master image pointing to the master tag, for easier development. I realize this doesn't scale to multiple devs, but I'm not sure specifying a version helps. Can we leave this out and figure out a better mechanism outside this PR?

Reply:
Absolutely, closing this down for the time being.